src/dashboard/src/Layout/NotificationBox.tsx

```diff
@@ -30,29 +26,44 @@ const usePaperStyle = makeStyles(theme => createStyles({
   }
 }))
 
-const NotificationBox: FunctionComponent<BoxProps> = (props) => {
-  const { notifications } = useContext(ConfigContext)
-  const { currentTeamId } = useContext(TeamContext)
-
-  const notification = useMemo(() => {
-    const notification = get(notifications, [currentTeamId])
-    if (notification === undefined) {
-      return get(notifications, ['.default'])
-    }
-    return notification
-  }, [notifications, currentTeamId])
-
+const ClusterNotificationBox: FunctionComponent<{ cluster: any }> = ({ cluster }) => {
+  const { data } = useFetch(`/api/clusters/${cluster.id}`, [cluster.id])
+  const notifications = useMemo(() => {
+    if (data === undefined) return []
+    if (!Array.isArray(data.notifications)) return []
+    return data.notifications as string[]
+  }, [data])
   const paperStyle = usePaperStyle()
+  return (
+    <>
+      {
+        notifications.map((notification, index) => (
+          <Fragment key={index}>
+            <Paper elevation={0} classes={paperStyle}>
+              <Info fontSize="small" color="primary"/>
+              <Typography
+                variant="body2"
+                component={Box}
+                flex={1}
+                paddingLeft={1}
+                dangerouslySetInnerHTML={{ __html: notification }}
+              />
+            </Paper>
+            <Divider/>
+          </Fragment>
+        ))
+      }
+    </>
+  )
+}
 
-  if (notification == null || notification === '') return null
-
+const NotificationBox: FunctionComponent<BoxProps> = (props) => {
+  const { clusters } = useContext(ClustersContext)
   return (
     <Box {...props}>
-      <Paper elevation={0} classes={paperStyle}>
-        <Info fontSize="small" color="primary"/>
-        <Typography variant="body2" component={Box} flex={1} paddingLeft={1}>{notification}</Typography>
-      </Paper>
-      <Divider/>
+      {clusters.map(cluster => (
+        <ClusterNotificationBox key={cluster.id} cluster={cluster}/>
```

Comment:
Not a hard change request: I think `ClusterNotifications` is better then `ClusterNotificationBox`.

The `NotificationBox` is actually a `Box` (see line 63) but the `ClusterNotificationBox` is not.

Reply:
> Not a hard change request: I think `ClusterNotifications` is better then `ClusterNotificationBox`.
> 
> The `NotificationBox` is actually a `Box` (see line 63) but the `ClusterNotificationBox` is not.

I see, thanks. I need to be more formal.